thoth/common/openshift.py

```diff
@@ -1608,6 +1608,35 @@ def get_workflow(
 
         return wf
 
+    def get_workflow_status_report(
+        self,
+        workflow_id: Optional[str] = None,
+        label_selector: Optional[str] = None,
+        namespace: Optional[str] = None,
+    ) -> Dict[str, Any]:
+        """Get workflow status report, derived from workflow status."""
+        if workflow_id is None:
+            return {"error": "No workflow id provided"}
+        try:
+            wf_status = self.get_workflow_status(
+                name=workflow_id, label_selector=label_selector, namespace=namespace
+            )
+            status = {}
+            # the Job has not started (yet) but the Workflow has been submitted
+            # if the Workflow fails, the status will contain the finished
+            # time of the workflow
+            status["status"] = {
```

Comment:
Can we return directly status? I think making a nested dict does not make sense in this case.

Reply:
I agree, will change that